ci_example_python/doc/python_code_guidelines.md

```diff
@@ -22,7 +22,15 @@ good practice to check them upon creation of a new package or code refactoring.
 
 ## II. Python Coding Guidelines
 
-Regarding the style, follow [PEP 8](https://www.python.org/dev/peps/pep-0008/).
+- Regarding the style, follow [PEP 8](https://www.python.org/dev/peps/pep-0008/).
+- Use [flake8](https://flake8.pycqa.org) and fix all issues it shows.  This will
+  ensure compliance with PEP 8 and also point out some critical issues like
+  usage of undefined variables.
```

Comment:
Maybe offer the procedure or tutorials link to install it on vscode?
As this editor is getting widely used in the MPI.

Reply:
I was already thinking about adding a list of links to plugins for popular editors (I would not limit it to VSCode).  I see two options:

1. Have another markdown file here for this and link to it.
2. Have a page on the github wiki of this repo for it and link to it.

Pro of 1. is to have everything together in one place, pro of 2. is that it would be easier for people to add plugins if they find something (I doubt that many would go through the hassle of making a PR for this).

Reply:
I added a list in a separate markdown file, please extend it, if you know more plugins.